packages/wrangler/package-lock.json

```diff
@@ -0,0 +1,5356 @@
+{
```

Comment:
Looks like you ran `npm install` inside the `packages/wrangler` directory.
Please can you delete this file.
The correct way to add a dependency to wrangler is to run `npm` from the top level and target its workspace:

```sh
npm install -w wrangler @sentry/node
```